### ISSUE TITLE
example for how to login_with_google

### ISSUE THREAD

**Issue by deanmalmgren** [Doist/todoist-python#15]
Do you have an example somewhere for how to use the `login_with_google` functionality?

I created my todoist account with my google credentials but I can't seem to login with the python wrapper for the life of me.

Thanks!

**Comment by eleftherios-c** [comment]
You basically provide 2 arguments your `email` and the `oauth2_token` that authenticates you to our server.  There is a lot of information on [how to generate your OAuth2 token](https://www.google.gr/search?q=generate+oauth2+token), but basically it boils to this:

1) Go to the [Google OAuth 2.0 Playground](https://developers.google.com/oauthplayground/)
2) Select `Google OAuth2 API v2` and then `https://www.googleapis.com/auth/userinfo.email`, and finally click to the `Authorize APIs` button
3) You get the `Authorization code`, and then you click `Exchange authorization code for tokens` button
4) You get a `Refresh token` and an `Access token`, and the `Access token` is what you are looking after

And here's an example with the Python module:

```python
import todoist
api = todoist.TodoistAPI()
api.login_with_google('your-email-address', 'your-oauth2-access-token')
...
```

**Comment by deanmalmgren** [comment]
Thanks for the post. Since I'm just trying to get at my own data, I finally realized that I could just use the API Token to initialize the `TodoistAPI` instance with

```
api = todoist.TodoistAPI(token='my-token')
```

I realize there's a lot of _information_ out there on how to generate OAuth2 tokens, but that doesn't necessarily make it _useful_. The steps you mention are definitely helpful because I didn't know which `scope` to use, for example. In case anyone else stumbles across this, I ended up using [oauth2client](https://oauth2client.readthedocs.io)'s [run_flow](https://developers.google.com/api-client-library/python/guide/aaa_oauth#command-line-tools) functionality to create a server-side oauth2 token. I was still having trouble with scopes, but I suspect the `userinfo.email` would have helped me actually obtain a working token.

I'll mark this as closed, but I suspect it may be helpful to add a few examples to the documentation for how to use OAuth2.
